jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -403,7 +403,6 @@ private void writeAnnotations(String inPath, String outFile) throws IOException
         "invalid input path - " + inPath);
     LOG(DEBUG, "DEBUG", "Writing Annotations to " + outFile);
 
-    new File(outFile).getParentFile().mkdirs();
```

Comment:
Why remove this? We want to make sure the path to this file exists, don't we? (Otherwise we need an explicit call to `mkdir -p` in e.g. buck, before running this tool)

Reply:
Good catch. Added it back.